extensions/eclipse-che-theia-plugin-ext/src/node/che-plugin-service.ts

```diff
@@ -383,7 +351,7 @@ export class ChePluginServiceImpl implements ChePluginService {
                 if (foundIndex >= 0) {
                     plugins.splice(foundIndex, 1);
                 } else {
-                    workspace.devfile.components.splice(workspace.devfile.components.indexOf(component), 1);
+                    workspace.devfile!.components!.splice(workspace.devfile!.components!.indexOf(component), 1);
```

Comment:
Could you please explain to me why `components` always exist? (It is more about the ` workspace.devfile.components!.forEach((component: cheApi.workspace.devfile.Component) => {` line above (342-nd at the moment of review).

Reply:
I added a verification for the `components` and its initialization if it's no exist.